frontend/src/cards/ui/Sources.tsx

```diff
@@ -149,25 +147,32 @@ export function Sources(props: SourcesProps) {
           <p className={styles.FootnoteTextNH}>Note. NH: Non-Hispanic. </p>
         )}
       </Grid>
-      {/* Sources inline with card action buttons */}
-      <Grid item xs={8} sm={9} md={10} container alignItems={'center'}>
-        {sourcesInfo}
-      </Grid>
+      {props.isMulti ?
```

Comment:
i think we should store the state of the multimap view in the URL bar, and then the user can link directly to an opened multimap just like any other view. im ok merging this first and then adding that as a follow up feature, or holding on this and getting that working as well, up to you

Reply:
but then the multimap modal could have a normal 3-dot menu just like every other card gets